react-web/src/pages/testHistory/TestHistory.tsx

```diff
@@ -5,19 +5,26 @@ import Button from "components/Button/Button";
 import "./TestHistory.scss";
 import { fetchData } from "api/api";
 
+interface ICampaign {
+  "certificateCreatedAt": string,
```

Comment:
my bad here, this property (`certificateCreatedAt `) doesn't exist anymore in the `ICampaing` record.
Now it belongs to the Certification object retrieved to its separate API route

Reply:
Removed